### PR TITLE
[CM-1007] Changed notification icon color

### PR DESCRIPTION
**Pull request by pahadi777**
Changed notification icon color

### CODE REVIEW

File: kommunicate/src/main/java/com/applozic/mobicomkit/api/notification/NotificationService.java [hunk 1]
```diff
@@ -71,6 +72,8 @@ public class NotificationService {
     private String notificationFilePath;
     public static final String BADGE_COUNT = "BADGE_COUNT";
     public static final String NO_ALERT = "NO_ALERT";
+    private final String notificationIconColor;
+    private final String DEFAULT_NOTIFICATION_ICON_COLOR = "3F51B5";
```
Comment (amntoppo) [inline kommunicate/src/main/java/com/applozic/mobicomkit/api/notification/NotificationService.java:76]: Don't give a default value. 
If Metadata is not present then don't set the color.

File: kommunicate/src/main/java/com/applozic/mobicomkit/api/notification/NotificationService.java [hunk 2]
```diff
@@ -426,10 +432,12 @@ public void notifyUserForNormalMessage(Contact contact, Channel channel, Message
 
         NotificationCompat.Builder mBuilder = new NotificationCompat.Builder(context, notificationChannels.getDefaultChannelId(muteNotifications(index)));
 
-        mBuilder.setSmallIcon(notificationInfo.smallIconResourceId)
+        mBuilder
+                .setSmallIcon(notificationInfo.smallIconResourceId)
```
Comment (amntoppo) [inline kommunicate/src/main/java/com/applozic/mobicomkit/api/notification/NotificationService.java:436]: remove extra line

**Review by amntoppo — CHANGES_REQUESTED** [review]

File: kommunicate/src/main/java/com/applozic/mobicomkit/api/notification/NotificationService.java [hunk 3]
```diff
@@ -6,15 +6,18 @@
...
 import android.net.Uri;
 import android.os.Build;
 
 import androidx.core.app.NotificationCompat;
 import androidx.core.app.NotificationManagerCompat;
+import androidx.core.content.ContextCompat;
 
 import android.text.Html;
 import android.text.TextUtils;
+import android.util.Log;
```
Comment (amntoppo) [inline kommunicate/src/main/java/com/applozic/mobicomkit/api/notification/NotificationService.java:20]: remove unused imports.

**Review by amntoppo — APPROVED** [review]

**Review by Sathyan-Elangovan — APPROVED** [review]
Add Changelog entry

**Review by Sathyan-Elangovan — CHANGES_REQUESTED** [review]
in Title add [UnReleased] instead mentioning the version. Because we have not released this version right

**Review by Sathyan-Elangovan — APPROVED** [review]

**Pull request merged**